exercises/transpose/example.lua

```diff
@@ -1,7 +1,7 @@
 local function Grid(s)
   local width = 0
   local rows = {}
-  for row in s:gmatch('([^\n]*)\n?') do
```

Comment:
Actually not, i just think in case of 5.3.1 we have bug which was fixed later.
If first version works for both, probably, it is better.